### PR TITLE
[ticket/13988] Use absolute path for image attachments in feeds

### PR DESCRIPTION
**Pull request by Zoddo**
Previously, atom feeds use relative links for image attachments

https://tracker.phpbb.com/browse/PHPBB3-13988

### CODE REVIEW

**Comment by Nicofuma** [comment]
You need to update a few tests

File: tests/functional/feed_test.php [hunk 1]
```diff
@@ -1395,7 +1395,7 @@ protected function assert_feed($params, $data)
 					foreach ($attachments as $i => $attachment)
 					{
 						$content = $crawler->filterXPath("//entry[{$entry_id}]/content")->text();
-						$url = "./download/file.php?id={$attachment['id']}";
+						$url = "http://localhost/download/file.php?id={$attachment['id']}";
```
Comment (CHItA) [inline tests/functional/feed_test.php]: Please take it into account that functional tests could be runned on different paths than `http://localhost` by defining `$phpbb_functional_url`

Comment (Zoddo) [inline tests/functional/feed_test.php]: Last commit amended :)

**Comment by CHItA** [comment]
:+1:

**Comment by Zoddo** [comment]
It's good?

**Pull request merged**